test/markdown.jl

```diff
@@ -1079,3 +1079,17 @@ t = """
     :-- | --:
     1   |   2"""
 @test sprint(Markdown.term, Markdown.parse(t), 0) == "a b\n– –\n1 2\n"
+
+# test Base.copy
+let
+  md = doc"test"
```

Comment:
Should be four space indents

Reply:
omg, I always forget it's four space indent in Base.
Can we have something like [EditorConfig](http://editorconfig.org/)?